downstream/modules/platform/proc-create-keycloak-client.adoc

```diff
@@ -2,14 +2,16 @@
 
 = Creating a client
 
-Use the following procedure to create the Client.
+Clients are entities that can request Keycloak to authenticate a user.
+Clients can also be entities that request identity information or an access token to securely invoke other services on the network that are secured by Keycloak.
```

Comment:
Keycloak clients authenticate Hub users with RHSSO.
Hub does not directly message or request RHSSO when a user authenticates. It goes through the Keycloak client. Once RHSSO validates or issues the OAuth token, the client provides the response to Hub and the user can log in.